make/tools.mk

```diff
@@ -16,7 +16,7 @@
 # Set up ARM (STM32) SDK
 ARM_SDK_DIR := $(TOOLS_DIR)/gcc-arm-none-eabi-5_4-2016q2
 # Checked below, Should match the output of $(shell arm-none-eabi-gcc -dumpversion)
-GCC_REQUIRED_VERSION := 5.4.1
+GCC_REQUIRED_VERSION := 5.2.1
```

Comment:
I install v5.2.1 . I have to change  these files when making the target. I forgot to change back before  push to github